holochain.tex

```diff
@@ -154,19 +154,27 @@ \subsection{Data-Centric and Agent-Centric Systems}
 
 The first point establishes the central aspect of Bitcoin's (and Blockchain applications' in general) strategy for solving or avoiding problems otherwise encountered in decentralized systems, and that is by trying to maintain a network state in which all nodes \textbf{should} have the same (local) chain.
 
-By contrast, for \sgit there is no such constraint on any $\chain_n$, $\chain_m$ in nodes $n$ and $m$ matching, also $V(e,v)$ only checks the structural validity of $e$ as a commit object not it's content, and $I(t)$ can be understood to be the set of git commands available to the user, and $\tau_\chain$ the function that adds a commit object and $\tau_\textrm{D}$ the function that adds code to the \texttt{index} triggered by \texttt{add}. $E$ is, similarly to \sbtc, not formally defined for \sgit. \todo{TODO: more detailed comparison with git?}
+By contrast, for \sgit there is no such constraint on any $\chain_n$, $\chain_m$ in nodes $n$ and $m$ matching, as git's core intent is to allow different agents act autonomously and divergently on a shared code-base, which would be impossible if the states always had to match.   
 
-Our model of a distributed system makes one thing very clear about the difference between \sbtc and \sgit.  As the size of $\mathcal{X}_n$ grows, necessarily all nodes of \sbtc must grow in size, whereas this is not necessarily the case for \sgit.  Though this seems like a clear limitation, it comes as a direct consequence of the constraint of $\forall n,m \in N: \chain_n\eqbang\chain_m$. Interestingly this is actually considered Bitcoin's strength as it defines the heart of ``consensus" in the Blockchain world, i.e., a limitation and method to the achieve sameness of state of all nodes.
+Through the lens of the formalisim the some other aspects of \sgit can be understood as follows:
+\begin{enumerate}
+\item the validation function $V(e,v)$ by default only checks the structural validity of $e$ as a commit object not it's content (though note that git does also support signing of commits which is also part of the validation)
+\item the stimulus function $I(t)$ for \sgit consists of the set of git commands available to the user
+\item the state transition function $\tau_\chain$ is the internal git function that adds a commit object and $\tau_\textrm{D}$ is the git function that adds code to the \texttt{index} triggered by \texttt{add}
+\item $E$ is, similarly to \sbtc, not formally defined for \sgit.
+\end{enumerate} 
+
+We leave a more in depth application of the formalism to \sgit as an excercise for the reader, however we underscore that the core difference between  \sbtc and \sgit lies in the formers constraint of $\forall n,m \in N: \chain_n\eqbang\chain_m$.  One direct consequence of this for \sbtc is that as the size of $\mathcal{X}_n$ grows, necessarily all nodes of \sbtc must grow in size, whereas this is not necessarily the case for \sgit, and in it lies the core of Bitcoin's scalability issues.
```

Comment:
Several missing backslashes at the end of commands lead to missing spaces after math symbols.